warehouse/templates/search/results.html

```diff
@@ -44,50 +46,9 @@ <h3 class="package-snippet__title">
 {%- endmacro %}
 
 
-{% macro pagination(page) -%}
-{% if page.items and page.page_count > 1 %}
-  {% set map = page.link_map() %}
-  <div class="button-group button-group--pagination">
-    {# previous button #}
-    {% if map.current_page.value == map.first_page.number %}
-    <a class="button button-group__button button--disabled">Previous</a>
-    {% else %}
-    <a href="{{ map.previous_page.href }}" class="button button-group__button">Previous</a>
-    {% endif %}
-
-    {# show first page button - special case #}
-    {% if map.current_page.value > map.first_page.number + map.radius %}
-    <a href="{{ map.first_page.href }}" class="button button-group__button">{{ map.first_page.number }}</a>
-    {% endif %}
-
-    {% for p in map.range_pages %}
-      {% if p.type == 'current_page' %}
-      <span class="button button-group__button button--primary">{{ p.number }}</span>
-      {% elif p.type == 'page' %}
-      <a href="{{ p.href }}" class="button button-group__button">{{ p.number }}</a>
-      {% elif p.type == 'span' %}
-      <span class="button button-group__button">...</span>
-      {% endif %}
-    {% endfor %}
-
-    {# show last page button - special case #}
-    {% if map.current_page.value < map.last_page.number - map.radius %}
-    <a href="{{ map.last_page.href }}" class="button button-group__button">{{ map.last_page.number }}</a>
-    {% endif %}
-
-    {# "next" button #}
-    {% if map.current_page.value == map.last_page.number %}
-    <a class="button button-group__button button--disabled">Next</a>
-    {% else %}
-    <a href="{{ map.next_page.href }}" class="button button-group__button">Next</a>
-    {% endif %}
-  </div>
-{% endif %}
-{%- endmacro %}
-
 {% block search_form_extra_inputs -%}
   {%- for classifier in applied_filters %}
-    <input name="c" type="hidden" value="{{ classifier|e }}">
+  <input name="c" type="hidden" value="{{ classifier|e }}">
```

Comment:
An unnecessary change.

Reply:
I thought so, too, but the linter failed on this.  I changed it back, and it looks like the Travis lint job fails, too.  Thanks for taking a look!